test/infrastructure/inmemory/api/v1alpha1/inmemorycluster_types.go

```diff
@@ -23,8 +23,9 @@ import (
 )
 
 const (
-	// ResourceGroupAnnotationName tracks the name of a resource group a InMemoryCluster cluster is linked to.
-	ResourceGroupAnnotationName = "inmemorycluster.infrastructure.cluster.x-k8s.io/resource-group"
+	// ListenerAnnotationName tracks the name of a resource group a InMemoryCluster cluster is linked to.
```

Comment:
```suggestion
	// ListenerAnnotationName tracks the name of a resource group a InMemoryCluster cluster is linked to.
```
"of a resource group" seems outdated now